UI/window-basic-main-outputs.hpp

```diff
@@ -1,15 +1,17 @@
 #pragma once
 
+#include <future>
 #include <memory>
 #include <string>
 
 #include <QFuture>
```

Comment:
Can this QFuture be removed?

Reply:
Given that the blame points to https://github.com/obsproject/obs-studio/commit/c8950900c36fd6d0bd014eb6fd24da829a63033d (#10633), it should be safe to remove.

Reply:
removed this and a `QPromise` include elsewhere too